README.md

```diff
@@ -396,7 +396,7 @@ RuntimeError: no running event loop
 ```
 
 What's happening here is that we are calling `rust_sleep` _before_ the future is
-actually running on the event loop created by `asyncio.run`. This is counter-intuitive, but expected behaviour, and unfortunately there doesn't seem to be a good way of solving this problem within PyO3 Asyncio itself.
```

Comment:
`behaviour` does not necessarily have to be changed to `behavior` - the preference between these depends on the preferred English variant.

Reply:
Thank you for your review! I totally misunderstood it as a typo 😅